src/trading-api/next/v1/bullish-trading-api.yml

```diff
@@ -3077,27 +3077,56 @@ components:
     TradingAccountResponse:
       type: object
       required:
-        - tradingAccountId
-        - tradingAccountName
-        - tradingAccountDescription
+        - isBorrowing
+        - isLending
         - isPrimaryAccount
+        - makerFee
+        - maxInitialLeverage
         - rateLimitToken
+        - takerFee
+        - tradingAccountDescription
+        - tradingAccountId
+        - tradingAccountName
       properties:
+        isBorrowing:
+          description: Whether the trading account is borrowing a loan
```

Comment:
Whether the trading account is borrowing

Reply:
We try to use the term **loan** only when talking about lending.

Reply:
THanks.

I removed the word loan.